src/main/resources/templates/user/create.html

```diff
@@ -0,0 +1,20 @@
+<!--<!DOCTYPE html>-->
+<!--<html lang="en" xmlns:th="http://www.thymeleaf.org/">-->
+<!--<head th:replace="fragments :: head"></head>-->
+<!--<body class="text-center ">-->
+<!--<nav th:replace="fragments :: navigation"></nav>-->
+<!--<div class="container-fluid">-->
+<!--    <form method="post">-->
+<!--        <div class="form-group">-->
+<!--            <lable>Name-->
+<!--                <input th:field="${user.name}" class="form-control">-->
+<!--            </lable>-->
+<!--            <span th:errors="${user.name}" class="error"></span>-->
+<!--        </div>-->
+<!--        <input type="submit" value="Create" class="btn btn-primary">-->
+<!--    </form>-->
+<!--</div>-->
+<!--<footer th:replace="fragments :: footer"></footer>-->
+<!--<div th:replace="fragments :: bootstrap-javascript"></div>-->
+<!--</body>-->
+<!-- </html>-->
```

Comment:
You can definitely go ahead and delete this file! It'll simplify what we're checking in the pull request.